gtfs/retriever/gtfs_retrieve.py

```diff
@@ -114,22 +116,23 @@ def parse_config(config_file_name):
     return config_dict
 
 
-def upload_gtfs_file_to_s3_bucket(connection, file_name, force=False):
+def download_file_and_upload_to_s3_bucket(connection, remote_file_name, force=False):
     """ download gtfs zip file from mot, and upload to s3 Bucket """
-    # file_name = get_utc_date() + '.zip'
-    tmp_file = '/tmp/' + file_name
+    filename = os.path.splitext(remote_file_name)[0] + datetime.datetime.now().strftime(
+        '-%Y-%m-%dT%H-%M-%S') + '.zip'
```

Comment:
- underscore to separate file name from date (instead of hyphen)
- files don't have to be .zip (they all are now, but future files might not be)

Reply:
instead of splitting joining, you can just add the date as a prefix to the original file name